hlx503Sup/hlx503.st

```diff
@@ -104,23 +105,24 @@ ss recondense {
             PVPUT(part_success, 0);
             // Set status that we are on part 1 of a recondense
             PVPUTSTR(part, "PART 1");
-            // Set heater to manual, 0% and temp setpoint to 0K TODO check this 
-            PVPUT(heater_off, 1);
+            // Set heater to manual, 0%
+            PVPUT(heater_off, 1, SYNC);
             // Store PID settings to reapply post-recondense
             PVPUTSTR(old_adjust_pids, adjust_pids_sp);
-            PVPUTSTR(old_autopid, autopid);
+            PVPUTSTR(old_autopid, autopid, SYNC);
             PVPUT(old_p, p_sp);
             PVPUT(old_i, i_sp);
             PVPUT(old_d, d_sp);
             // Use PIDs from PVs not tpar files or autopid
             PVPUTSTR(adjust_pids_sp, "NO");
-            PVPUTSTR(autopid_sp, "OFF");
+            PVPUTSTR(autopid_sp, "OFF", SYNC);
             // Set PIDs
             PVPUT(p_sp, recondense_sorb_p);
             PVPUT(i_sp, recondense_sorb_i);
             PVPUT(d_sp, recondense_sorb_d);
             // Set temperature then heater mode to auto
-            PVPUT(temp_sorb_sp, recondense_sorb_temp_sp);
+			PVPUT(heater_off, 1, SYNC);
+            PVPUT(temp_sorb_sp, recondense_sorb_temp_sp, SYNC);
```

Comment:
Why do we need `SYNC` on some writes and not others (e.g. `p_sp` above, or why we *don't* need it when we write again to `temp_sorb_sp` below)? (can you document the reasoning in a comment or on the wiki)

Reply:
I think we tried this before using the throttle, I will remove them. Weirdly they came up as compiler warnings with the SYNC flag so not entirely sure they worked anyway.